composer.json

```diff
@@ -6,7 +6,8 @@
     "type": "project",
     "require": {
         "php": ">=5.5.9",
```

Comment:
If I'm not mistaking, there was no requirements to use php 7, but ok, no problems for me